### ISSUE TITLE
Pyright exception on ClientConnectorSSLError

### ISSUE THREAD

**Issue by woutercoppens** [microsoft/pyright#4580]
**Describe the bug**

```python
import aiohttp
...

       except (aiohttp.ClientConnectorSSLError) as exception: 
            # Expired certificate / Date ISSUE
            # pylint: disable=bad-exception-context
            raise OpenMoticsConnectionSslError(
                "Error with SSL certificate."
            ) from exception
```
```console
pyright 1.1.263
/localgateway.py
  /localgateway.py:157:17 - error: "Type[ClientConnectorSSLError]" is not valid exception class
  /localgateway.py:162:20 - error: Expected exception object, exception class or None
    "ClientConnectorSSLError" does not derive from BaseException
```

**To Reproduce**
pip install pyright
pyright

**Expected behavior**
Is there a way to let pyright ignore this error? 

If your code relies on symbols that are imported from a third-party library, include the associated import statements and specify which versions of those libraries you have installed.

**VS Code extension or command-line**
command-line

**Additional context**
See also https://github.com/aio-libs/aiohttp/issues/6845

**Comment by erictraut** [comment]
The `ClientConnectorSSLError` class is defined using a dynamic expression for its base classes. Pyright should handle that case in a more elegant manner. I've updated the logic, which eliminates this false positive error. The change will be included in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.293, which I just published. It will also be included in a future release of pylance.
